packages/hash/integration/src/tests/model/link.model.test.ts

```diff
@@ -244,114 +225,79 @@ describe("Link model class ", () => {
       createdByAccountId,
       source: entityA,
       destination: entityB,
-      stringifiedPath: "$.test",
+      stringifiedPath,
       index: 0,
     });
 
-    await entityA.refetchLatestVersion(db);
-
-    const entityAVersionId2 = entityA.entityVersionId;
-
-    expect(entityAVersionId1).not.toBe(entityAVersionId2);
+    const entityATimestamp2 = new Date();
 
     const linkAToC = await Link.create(db, {
       createdByAccountId,
       source: entityA,
       destination: entityC,
-      stringifiedPath: "$.test",
+      stringifiedPath,
       index: 0,
     });
 
-    await entityA.refetchLatestVersion(db);
-
-    const entityAVersionId3 = entityA.entityVersionId;
-
-    expect(entityAVersionId2).not.toBe(entityAVersionId3);
+    const entityATimestamp3 = new Date();
 
     await entityA.deleteOutgoingLink(db, {
       linkId: linkAToC.linkId,
       deletedByAccountId: accountId,
     });
 
-    const entityAVersionId4 = entityA.entityVersionId;
-
-    expect(entityAVersionId3).not.toBe(entityAVersionId4);
-
-    const entityAVersion1 = (await Entity.getEntity(db, {
-      accountId,
-      entityVersionId: entityAVersionId1,
-    }))!;
+    const entityATimestamp4 = new Date();
 
-    expect(entityAVersion1).not.toBe(null);
-
-    const entityAVersion1OutgoingLinks = await entityAVersion1.getOutgoingLinks(
-      db,
-    );
-
-    expect(entityAVersion1OutgoingLinks.length).toBe(0);
-
-    const entityAVersion2 = (await Entity.getEntity(db, {
-      accountId,
-      entityVersionId: entityAVersionId2,
-    }))!;
-
-    expect(entityAVersion2).not.toBe(null);
-
-    const entityAVersion2OutgoingLinks = await entityAVersion2.getOutgoingLinks(
-      db,
-    );
+    const entityATimestamp1OutgoingLinks = await entityA.getOutgoingLinks(db, {
+      activeAt: entityATimestamp1,
+      stringifiedPath,
+    });
 
-    expect(entityAVersion2OutgoingLinks.length).toBe(1);
-    expect(entityAVersion2OutgoingLinks[0].linkId).toBe(linkAToB.linkId);
-    expect(entityAVersion2OutgoingLinks[0].index).toBe(0);
+    expect(entityATimestamp1OutgoingLinks.length).toBe(0);
 
-    const entityAVersion3 = (await Entity.getEntity(db, {
-      accountId,
-      entityVersionId: entityAVersionId3,
-    }))!;
+    const entityATimestamp2OutgoingLinks = await entityA.getOutgoingLinks(db, {
+      activeAt: entityATimestamp2,
+      stringifiedPath,
+    });
 
-    expect(entityAVersion3).not.toBe(null);
+    expect(entityATimestamp2OutgoingLinks.length).toBe(1);
```

Comment:
These are called `entityATimestampNOutgoingLinks` where N is just the timestamp and not the number of outgoing links, right?

It was just slightly confusing to see the length being 1 when  it says `2OutgoingLinks`, but i assume it's `Timestamp2`. Just want to make sure here :)

Reply:
Yeah, those are supposed to be all the outgoing links at `Timestamp2`

Reply:
`TimestampA`, `B` and `C` might've been less confusing - not sure if it's worth making a change?

Reply:
No, was just making sure. Thank you!